src/resharper/run.ps1

```diff
@@ -3,10 +3,19 @@ param(
     [string] $SolutionDirectoryPath,
 
     [Parameter(Mandatory=$True)]
-    [string] $OutputDirectoryPath,
+    [string] $SolutionRelativePath,
 
     [Parameter(Mandatory=$True)]
-    [string] $SolutionRelativePath
+    [string] $OutputDirectoryPath,
+
+    [Parameter(Mandatory=$False)]
+    [string] $NugetPackagesSource = $null,
+
+    [Parameter(Mandatory=$False)]
+    [string] $NugetPackagesSourceUserName = $null,
+
+    [Parameter(Mandatory=$False)]
+    [SecureString] $NugetPackagesSourcePassword = $null
```

Comment:
This is an old code that I forgot to remove. Please use `NugetConfigFilePath` instead of these parameters